cirq/ops/pauli_string_test.py

```diff
@@ -1558,3 +1558,23 @@ def test_deprecated():
         _ = cirq.PauliString({
             a: 'x'
         }).expectation_from_state_vector(state=state_vector, qubit_map={a: 0})
+
+
+def test_circuit_diagram_info():
+    a, b, c = cirq.LineQubit.range(3)
+
+    assert cirq.circuit_diagram_info(cirq.PauliString(), default=None) is None
+
+    cirq.testing.assert_has_diagram(
+        cirq.Circuit(
+            cirq.PauliString({a: cirq.X}),
+            -cirq.PauliString({a: cirq.X}),
+            cirq.X(a) * cirq.Z(c),
+            1j * cirq.X(a) * cirq.Y(b),
+        ), """
```

Comment:
Cover the case outside the Pauli group, e.g. `cirq.PauliString({a: cirq.X}, coefficient=np.sqrt(1j))`.

Reply:
Done.